generators/container/templates/container.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './<%= _.kebabCase(componentName) %>.css';
+
+function mapStateToProps(state) {
+  return {
+
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+
+  };
+}
+
+@connect(mapStateToProps, mapDispatchToProps)
```

Comment:
Shouldn't we stay away from decorators while the syntax and semantics are adjusted?

The legacy transformator plugin exists because of existing projects that already depend on that behavior, but I'm not sure if it's a good idea to add that dependency to new projects.

Though I admit it is a visually nice way to create a container.